### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Tomer51m**
updated user.model.js
updated server.js
updated partially user.routes.js 

installed validator.js from npm - to perform email validation.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+app.use(cors({
+    origin:'http://localhost:3000'
+}));
+app.use(bodyParser.json());
+app.use(users.route);
+app.use(posts.route);
+app.use(comments.route);
+
+app.listen(8080);
```
Comment (UwebDevCom) [inline server/server.js]: move console.log to listen == app.listen(8080, ()=>console.log('server is running'))

File: server/src/users/users.route.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const { Router } = require ('express');
+const route = Router();
+
+route.get('/user', (req, res) => {
```
Comment (UwebDevCom) [inline server/src/users/users.route.js]: users

File: server/server.js (hunk 1 above)
Comment (Tomer51m) [reply server/server.js]: will do

File: server/src/users/users.route.js (hunk 2 above)
Comment (Tomer51m) [reply server/src/users/users.route.js]: done

File: server/server.js (hunk 1 above)
Comment (Tomer51m) [reply server/server.js]: boom 💥

File: server/server.js [hunk 3]
```diff
@@ -15,4 +14,6 @@ app.use(cors({
 app.use(bodyParser.json());
 app.use(users.route);
 
-app.listen(8081);
```
Comment (elazar91) [inline server/server.js]: looking good 👍

**Pull request merged**